docs/examples/google-cloud-function.ts

```
@@ -0,0 +1,27 @@
+import { Telegraf } from 'telegraf'
+
+const BOT_TOKEN = process.env.BOT_TOKEN ?? '<BOT TOKEN HERE>'
```

Please use empty string as default value or throw if it's `undefined`.